src/Markdown/ImageProcessor.php

```diff
@@ -0,0 +1,41 @@
+<?php
+/**
+ * This file is authored by PrestaShop SA and Contributors <contact@prestashop.com>
+ *
+ * It is distributed under MIT license, since 2022.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Help\PrestaShop\Markdown;
+
+use League\CommonMark\Event\DocumentParsedEvent;
+use League\CommonMark\Extension\CommonMark\Node\Inline\Image;
+
+class ImageProcessor
+{
+    private string $baseUrl;
+
+    public function __construct(string $baseUrl)
+    {
+        $this->baseUrl = $baseUrl;
+    }
+
+    public function __invoke(DocumentParsedEvent $event): void
+    {
+        foreach ($event->getDocument()->iterator() as $image) {
+            if (!($image instanceof Image)) {
+                continue;
+            }
+            $image->data->set('attributes', ['class' => 'img-responsive']);
+            if (!parse_url($image->getUrl(), PHP_URL_HOST)) {
```

Comment:
Can we inject `PHP_URL_HOST`

```
public function __construct(string $baseUrl, string $hosUrl = PHP_URL_HOST)
```

This makes unit testing easier for mocks

Reply:
I'm not sure to understand 🤔, `PHP_URL_HOST` is a "flag" to return the "host" part of an url when calling `parse_url` instead of returning an array with every url parts (scheme, host, port, etc...)

Reply:
Woups I thought it was a php constant similar to _CURRENT_DIR

Apologies